buy-shirt.spec.ts

```diff
@@ -0,0 +1,18 @@
+describe('Buy a t-shirt', () => {
+
+    it('then the t-shirt should be bought', () => {
+        cy.visit('http://automationpractice.com/')
+        cy.get('#block_top_menu > ul > li:nth-child(3) > a').click()
+        cy.get('#center_column a.button.ajax_add_to_cart_button.btn.btn-default').click()
+        cy.get('[style*="display: block;"] .button-container > a').click()
+        cy.get('.cart_navigation span').click()
+
+        cy.get('#email').type('aperdomobo@gmail.com')
+        cy.get('#passwd').type('WorkshopProtractor')
+
+        // Debes completar la prueba ...
```

Comment:
you must complete the test, using the gif in the documentation.

Reply:
Juan, I should open http://automationpractice.com/ from cypress when this opens google? I mean I try this but google open a list but when do click to http://automationpractice.com/ doesn't work

Reply:
I opened google since my google chrome navigator and work correctly :). What I should do?

Reply:
I finish the test! Finally :)

Reply:
I'm still waiting for your review and approval Pleaseeee! @renardete